yt_dlp/update.py

```diff
@@ -164,8 +164,8 @@ def release_name(self):
         label = _FILE_SUFFIXES[detect_variant()]
         if label:
             machine = platform.machine().lower()
-            if machine == 'aarch64' or machine.startswith("arm"):
-                label = f'_{machine}{label}'
+            if machine == 'aarch64' or machine.startswith('arm'):
```

Comment:
This should specifically look for `arm7l`. Any other architecture should give the "unofficial build" message

Reply:
Fixed in a0e827e